pkg/cloud/aws/athenaintegration.go

```diff
@@ -347,6 +353,15 @@ func (ai *AthenaIntegration) RowToCloudCost(row types.Row, aqi AthenaQueryIndexe
 		}
 	}
 
+	for _, awsColumnName := range aqi.AWSTagColumns {
+		// partially remove prefix leaving "aws_"
+		labelName := strings.TrimPrefix(awsColumnName, "resource_tags_")
```

Comment:
define new const, or string.TrimSuffix your existing const

Reply:
Could also just strip prefix in the first loop, and then append here.